src/edu/csus/ecs/pc2/ui/EditProblemPane.java

```diff
@@ -1654,13 +1655,13 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
      * 
      */
```

Comment:
I would add **javadoc** to the method comment to indicate what this method returns now.  eg.
`@return true if problem was successfully updated, false if not`